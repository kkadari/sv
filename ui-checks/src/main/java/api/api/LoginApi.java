package api.api;

import io.restassured.response.Response;

import static io.restassured.RestAssured.given;

public class LoginApi {

    public static Response login(String baseUrl, String username, String password){
        return given()
                .params("username", username, "password", password)
               .when()
                .post(baseUrl + "/cs_login");
    }
}
