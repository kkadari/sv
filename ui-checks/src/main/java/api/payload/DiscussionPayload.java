package api.payload;

public class DiscussionPayload {

//    {
//        "subject": "asdasdasads",
//        "markAsQuestion": false,
//        "body": "<body><p>asdasdasdasdasasdas</p></body>",
//        "handlingLevel": "2",
//        "publishBar": {
//            "visibility": "all"
//        },
//        "notAttributable": false,
//    }

    private String subject;
    private Boolean markAsQuestion;
    private String body;
    private String handlingLevel;
    private PublishPayload publishBar;
    private boolean notAttributable;

    public DiscussionPayload(String subject, Boolean markAsQuestion, String body, String handlingLevel, boolean notAttributable) {
        this.subject = subject;
        this.markAsQuestion = markAsQuestion;
        this.body = body;
        this.handlingLevel = handlingLevel;
        this.publishBar = new PublishPayload();
        this.notAttributable = notAttributable;
    }

    public String getSubject() {
        return subject;
    }

    public Boolean getMarkAsQuestion() {
        return markAsQuestion;
    }

    public String getBody() {
        return body;
    }

    public String getHandlingLevel() {
        return handlingLevel;
    }

    public PublishPayload getPublishBar() {
        return publishBar;
    }

    public boolean isNotAttributable() {
        return notAttributable;
    }
}
