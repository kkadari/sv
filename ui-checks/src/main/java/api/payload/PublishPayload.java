package api.payload;

public class PublishPayload {

    private String visibility;

    public PublishPayload() {
        this.visibility = "all";
    }

    public String getVisibility() {
        return visibility;
    }
}
