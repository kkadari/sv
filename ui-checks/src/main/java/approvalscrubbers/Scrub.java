package approvalscrubbers;

import java.util.ArrayList;

public class Scrub {

    public static String scrub(ArrayList<String> scrubbers, String content){
        for(String scrubber : scrubbers){
            content = content.replaceAll(scrubber, "***scrubbed***");
        }

        return content;
    }

}
