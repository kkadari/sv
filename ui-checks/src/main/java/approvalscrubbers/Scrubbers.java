package approvalscrubbers;

public class Scrubbers {

    public static String scrubJiveId(){
        return "id=\"jive-[0-9]*\"";
    }

    public static String scrubThreadRef(){
        return "\\/thread\\/[0-9]*";
    }

    public static String scrubDisplayObjectKeys(){
        return "name=\"displayedObjectKeys\" value=\"[0-9_,]*\"";
    }

}
