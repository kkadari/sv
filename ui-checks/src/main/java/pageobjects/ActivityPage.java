package pageobjects;

import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.support.FindBy;
import org.openqa.selenium.support.How;

import java.util.List;

public class ActivityPage extends PageObject {

    @FindBy(how = How.CSS, using = ".j-act-title")
    List<WebElement> divTitles;

    public ActivityPage(WebDriver driver){
        super(driver);
    }

    public String returnStreamListHtml(){
        String streams = "";

        for(WebElement title : divTitles){
            streams += title.getAttribute("outerHTML");
        }

        return streams;
    }

}
