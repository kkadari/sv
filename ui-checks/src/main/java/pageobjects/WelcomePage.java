package pageobjects;

import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.support.FindBy;
import org.openqa.selenium.support.How;
import org.openqa.selenium.support.ui.ExpectedConditions;
import org.openqa.selenium.support.ui.WebDriverWait;

public class WelcomePage extends PageObject {

    @FindBy(how = How.CSS, using = ".content-small .widget-container")
    WebElement divWidgetContainer;

    @FindBy(how = How.CSS, using = "#whats-happening")
    WebElement divWhatsHappening;

    @FindBy(how = How.CSS, using = "#feed-popular")
    WebElement listFeedPopular;

    @FindBy(how = How.CSS, using = "#feed-following")
    WebElement listFeedFollowing;

    @FindBy(how = How.CSS, using = ".event-content")
    WebElement divContent;

    @FindBy(how = How.CSS, using = ".footer-main")
    WebElement divFooter;

    @FindBy(how = How.CSS, using = "#customBrowseMenuAnchor")
    WebElement linkContentMenu;

    @FindBy(how = How.CSS, using = "#customBrowseMenu")
    WebElement divBrowseMenu;

    @FindBy(how = How.CSS, using = "#customBrowseMenu .j-pop-desc")
    WebElement divContentMenuItem;

    @FindBy(how = How.CSS, using = "#customMembersMenuAnchor")
    WebElement linkMembersMenu;

    @FindBy(how = How.CSS, using = "#customMembersMenu")
    WebElement divMembersMenu;

    @FindBy(how = How.CSS, using = "#navCommunitiesAndNodes")
    WebElement linkCommunityMenu;

    @FindBy(how = How.CSS, using = "#menuCommunitiesAndNodes")
    WebElement divCommunityMenu;

    @FindBy(how = How.CSS, using = "#menuCommunitiesAndNodes .j-pop-desc")
    WebElement divCommunityMenuItem;

    private WebDriverWait wait = new WebDriverWait(driver, timeout);

    public WelcomePage(WebDriver driver){
        super(driver);
    }

    public String returnWidgetContainerHtml(){
        return divWidgetContainer.getAttribute("outerHTML");
    }

    public String returnWhatsHappeningHtml(){
        wait.until(ExpectedConditions.visibilityOf(divContent));

        return divWhatsHappening.getAttribute("outerHTML");
    }

    public void clickFeedPopular() {
        wait.until(ExpectedConditions.visibilityOf(listFeedPopular));

        listFeedPopular.click();
    }

    public void clickFeedFollowing() {
        wait.until(ExpectedConditions.visibilityOf(listFeedFollowing));

        listFeedFollowing.click();
    }

    public String returnFooterHtml() {
        return divFooter.getAttribute("outerHTML");
    }

    public void clickContent() {
        linkContentMenu.click();
    }

    public String returnContentMenuPopupHtml(){
        wait.until(ExpectedConditions.visibilityOf(divContentMenuItem));

        return divBrowseMenu.getAttribute("outerHTML");
    }

    public void clickMembers() {
        linkMembersMenu.click();
    }

    public String returnMembersMenuPopupHtml() {
        return divMembersMenu.getAttribute("outerHTML");
    }

    public void clickCommunityAndNodes() {
        linkCommunityMenu.click();
    }

    public String returnCommunityAndNodesPopupHtml() {
        wait.until(ExpectedConditions.visibilityOf(divCommunityMenuItem));

        return divCommunityMenu.getAttribute("outerHTML");
    }

}
