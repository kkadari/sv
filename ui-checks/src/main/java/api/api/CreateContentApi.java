package api.api;

import api.payload.DiscussionPayload;
import io.restassured.http.ContentType;
import io.restassured.http.Cookies;

import static io.restassured.RestAssured.given;

public class CreateContentApi {

    public static void createDiscussion(Cookies cookies, String baseUrl, DiscussionPayload discussionPayload){
        given()
            .contentType(ContentType.JSON)
            .cookies(cookies)
            .body(discussionPayload)
        .when()
            .post(baseUrl + "/__services/v2/rest/discussion");
    }
}
