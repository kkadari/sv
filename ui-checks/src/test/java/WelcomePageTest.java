import api.api.CreateContentApi;
import api.payload.DiscussionPayload;
import approvalscrubbers.Scrub;
import approvalscrubbers.Scrubbers;
import org.approvaltests.Approvals;
import org.junit.Before;
import org.junit.Test;
import pageobjects.WelcomePage;

import java.util.ArrayList;

public class WelcomePageTest extends TestSetup {

    @Before
    public void generateTestContent(){
        for(int i = 1; i <= 7; i++){
            DiscussionPayload discussionPayload = new DiscussionPayload("Check discussion", false, "Generated for automation", "1", false);

            CreateContentApi.createDiscussion(cookies, baseUrl, discussionPayload);
        }
    }

    @Test
    public void approveStaticCallToAction(){
        driver.navigate().to(baseUrl + "/welcome");

        WelcomePage welcome = new WelcomePage(driver);
        String rawWidgetContainer = welcome.returnWidgetContainerHtml();

        ArrayList<String> scrubbers = new ArrayList<String>(){{
            add(Scrubbers.scrubJiveId());
        }};

        String scrubbedWidgetContainer = Scrub.scrub(scrubbers, rawWidgetContainer);

        Approvals.verifyHtml(scrubbedWidgetContainer);
    }

    @Test
    public void approveWhatsHappening(){
        driver.navigate().to(baseUrl + "/welcome");

        WelcomePage welcome = new WelcomePage(driver);
        String whatsHappeningContainer = welcome.returnWhatsHappeningHtml();

        ArrayList<String> scrubbers = new ArrayList<String>(){{
            add(Scrubbers.scrubJiveId());
            add(Scrubbers.scrubThreadRef());
            add(Scrubbers.scrubDisplayObjectKeys());
        }};

        String scrubbedWhatsHappeningContainer = Scrub.scrub(scrubbers, whatsHappeningContainer);

        Approvals.verifyHtml(scrubbedWhatsHappeningContainer);
    }

    @Test
    public void approveWhatsHappeningPopularTab(){
        driver.navigate().to(baseUrl + "/welcome");

        WelcomePage welcome = new WelcomePage(driver);
        welcome.clickFeedPopular();

        String whatsHappeningContainer = welcome.returnWhatsHappeningHtml();

        ArrayList<String> scrubbers = new ArrayList<String>(){{
            add(Scrubbers.scrubJiveId());
            add(Scrubbers.scrubThreadRef());
            add(Scrubbers.scrubDisplayObjectKeys());
        }};

        String scrubbedWhatsHappeningContainer = Scrub.scrub(scrubbers, whatsHappeningContainer);

        Approvals.verifyHtml(scrubbedWhatsHappeningContainer);
    }

    @Test
    public void approveWhatsHappeningFollowingTab(){
        driver.navigate().to(baseUrl + "/welcome");

        WelcomePage welcome = new WelcomePage(driver);
        welcome.clickFeedFollowing();

        String whatsHappeningContainer = welcome.returnWhatsHappeningHtml();

        ArrayList<String> scrubbers = new ArrayList<String>(){{
            add(Scrubbers.scrubJiveId());
            add(Scrubbers.scrubThreadRef());
            add(Scrubbers.scrubDisplayObjectKeys());
        }};

        String scrubbedWhatsHappeningContainer = Scrub.scrub(scrubbers, whatsHappeningContainer);

        Approvals.verifyHtml(scrubbedWhatsHappeningContainer);
    }

}
