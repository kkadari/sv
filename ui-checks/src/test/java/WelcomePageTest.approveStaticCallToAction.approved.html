<div class="widget-container" id="widget-2006">
	<div class="calls-to-action vertical">
    <div class="cta">
      <h2>Welcome to CiSP</h2>
      <div id="cisp_intro_video_poster"><img src="/___sbsstatic___/intro-video/cisp_intro_video_poster_sm.jpg"><div class="fa fa-play-circle-o play-icon" aria-hidden="true"></div></div>
      <div id="cisp_intro_video_container" class="jive-modal j-pop">
        <header class="j-modal-header"><h2>Welcome to CiSP</h2></header>
        <a href="#" class="j-modal-close-top close" title="Close Popup">Close<span class="j-close-icon j-ui-elem" role="img"></span></a>
        <section class="j-modal-content clearfix">
          <div poster="/___sbsstatic___/intro-video/cisp_intro_video_poster.jpg" preload="none" class="video-js vjs-default-skin vjs-paused vjs-controls-enabled vjs-workinghover vjs-user-active" id="cisp_intro_video" role="region" aria-label="video player"><video id="cisp_intro_video_html5_api" class="vjs-tech" preload="none" poster="/___sbsstatic___/intro-video/cisp_intro_video_poster.jpg" src="/___sbsstatic___/intro-video/cisp_intro_1080.mp4">
            <source src="/___sbsstatic___/intro-video/cisp_intro_1080.mp4" type="video/mp4" label="1080p" res="1080">
            <source src="/___sbsstatic___/intro-video/cisp_intro_720.mp4" type="video/mp4" label="720p" res="720">
            <source src="/___sbsstatic___/intro-video/cisp_intro_480.mp4" type="video/mp4" label="480p" res="480">
            <source src="/___sbsstatic___/intro-video/cisp_intro_240.mp4" type="video/mp4" label="240p" res="240">
          </video><div></div><div class="vjs-poster" tabindex="-1" style="background-image: url(&quot;/___sbsstatic___/intro-video/cisp_intro_video_poster.jpg&quot;);"></div><div class="vjs-text-track-display vjs-hidden" aria-live="assertive" aria-atomic="true"></div><div class="vjs-loading-spinner" dir="ltr"></div><button class="vjs-big-play-button" type="button" aria-live="polite"><span class="vjs-control-text">Play Video</span></button><div class="vjs-control-bar" dir="ltr" role="group"><button class="vjs-play-control vjs-control vjs-button " type="button" aria-live="polite"><span class="vjs-control-text">Play</span></button><div class="vjs-volume-menu-button vjs-menu-button vjs-menu-button-inline vjs-control vjs-button  vjs-volume-menu-button-horizontal vjs-vol-3" tabindex="0" role="button" aria-live="polite"><div class="vjs-menu"><div class="vjs-menu-content"><div tabindex="0" class="vjs-volume-bar vjs-slider-bar vjs-slider vjs-slider-horizontal" role="slider" aria-valuenow="0" aria-valuemin="0" aria-valuemax="100" aria-label="volume level"><div class="vjs-volume-level"><span class="vjs-control-text"></span></div></div></div></div><span class="vjs-control-text">Mute</span></div><div class="vjs-current-time vjs-time-control vjs-control"><div class="vjs-current-time-display" aria-live="off"><span class="vjs-control-text">Current Time</span> 0:00</div></div><div class="vjs-time-control vjs-time-divider"><div><span>/</span></div></div><div class="vjs-duration vjs-time-control vjs-control"><div class="vjs-duration-display" aria-live="off"><span class="vjs-control-text">Duration Time</span> 0:00</div></div><div class="vjs-progress-control vjs-control"><div tabindex="0" class="vjs-progress-holder vjs-slider vjs-slider-horizontal" role="slider" aria-valuenow="NaN" aria-valuemin="0" aria-valuemax="100" aria-label="progress bar" aria-valuetext="0:00"><div class="vjs-load-progress"><span class="vjs-control-text"><span>Loaded</span>: 0%</span></div><div class="vjs-mouse-display" data-current-time="0:00" style="left: 0px;"></div><div class="vjs-play-progress vjs-slider-bar" data-current-time="0:00" style="width: 0%;"><span class="vjs-control-text"><span>Progress</span>: 0%</span></div></div></div><div class="vjs-live-control vjs-control vjs-hidden"><div class="vjs-live-display" aria-live="off"><span class="vjs-control-text">Stream Type</span>LIVE</div></div><div class="vjs-remaining-time vjs-time-control vjs-control"><div class="vjs-remaining-time-display" aria-live="off"><span class="vjs-control-text">Remaining Time</span> -0:00</div></div><div class="vjs-custom-control-spacer vjs-spacer ">&nbsp;</div><div class="vjs-playback-rate vjs-menu-button vjs-menu-button-popup vjs-control vjs-button vjs-hidden" tabindex="0" role="menuitem" aria-live="polite" aria-expanded="false" aria-haspopup="true"><div class="vjs-menu" role="presentation"><ul class="vjs-menu-content" role="menu"></ul></div><span class="vjs-control-text">Playback Rate</span><div class="vjs-playback-rate-value">1</div></div><div class="vjs-chapters-button vjs-menu-button vjs-menu-button-popup vjs-control vjs-button vjs-hidden" tabindex="0" role="menuitem" aria-live="polite" aria-expanded="false" aria-haspopup="true" aria-label="Chapters Menu"><div class="vjs-menu" role="presentation"><ul class="vjs-menu-content" role="menu"><li class="vjs-menu-title" tabindex="-1">Chapters</li></ul></div><span class="vjs-control-text">Chapters</span></div><div class="vjs-descriptions-button vjs-menu-button vjs-menu-button-popup vjs-control vjs-button vjs-hidden" tabindex="0" role="menuitem" aria-live="polite" aria-expanded="false" aria-haspopup="true" aria-label="Descriptions Menu"><div class="vjs-menu" role="presentation"><ul class="vjs-menu-content" role="menu"><li class="vjs-menu-item vjs-selected" tabindex="-1" role="menuitemcheckbox" aria-live="polite" aria-checked="true">descriptions off<span class="vjs-control-text">, selected</span></li></ul></div><span class="vjs-control-text">Descriptions</span></div><div class="vjs-subtitles-button vjs-menu-button vjs-menu-button-popup vjs-control vjs-button vjs-hidden" tabindex="0" role="menuitem" aria-live="polite" aria-expanded="false" aria-haspopup="true" aria-label="Subtitles Menu"><div class="vjs-menu" role="presentation"><ul class="vjs-menu-content" role="menu"><li class="vjs-menu-item vjs-selected" tabindex="-1" role="menuitemcheckbox" aria-live="polite" aria-checked="true">subtitles off<span class="vjs-control-text">, selected</span></li></ul></div><span class="vjs-control-text">Subtitles</span></div><div class="vjs-captions-button vjs-menu-button vjs-menu-button-popup vjs-control vjs-button vjs-hidden" tabindex="0" role="menuitem" aria-live="polite" aria-expanded="false" aria-haspopup="true" aria-label="Captions Menu"><div class="vjs-menu" role="presentation"><ul class="vjs-menu-content" role="menu"><li class="vjs-menu-item vjs-selected" tabindex="-1" role="menuitemcheckbox" aria-live="polite" aria-checked="true">captions off<span class="vjs-control-text">, selected</span></li></ul></div><span class="vjs-control-text">Captions</span></div><div class="vjs-audio-button vjs-menu-button vjs-menu-button-popup vjs-control vjs-button vjs-hidden" tabindex="0" role="menuitem" aria-live="polite" aria-expanded="false" aria-haspopup="true" aria-label="Audio Menu"><div class="vjs-menu" role="presentation"><ul class="vjs-menu-content" role="menu"></ul></div><span class="vjs-control-text"></span></div><button class="vjs-fullscreen-control vjs-control vjs-button " type="button" aria-live="polite"><span class="vjs-control-text">Fullscreen</span></button></div><div class="vjs-error-display vjs-modal-dialog vjs-hidden " tabindex="-1" aria-describedby="cisp_intro_video_component_336_description" aria-hidden="true" aria-label="Modal Window" role="dialog"><p class="vjs-modal-dialog-description vjs-offscreen" id="cisp_intro_video_component_336_description">This is a modal window.</p><div class="vjs-modal-dialog-content" role="document"></div></div><div class="vjs-caption-settings vjs-modal-overlay vjs-hidden"><div class="vjs-tracksettings">
      <div class="vjs-tracksettings-colors">
        <div class="vjs-fg-color vjs-tracksetting">
            <label class="vjs-label">Foreground</label>
            <select>
              <option value="">---</option>
              <option value="#FFF">White</option>
              <option value="#000">Black</option>
              <option value="#F00">Red</option>
              <option value="#0F0">Green</option>
              <option value="#00F">Blue</option>
              <option value="#FF0">Yellow</option>
              <option value="#F0F">Magenta</option>
              <option value="#0FF">Cyan</option>
            </select>
            <span class="vjs-text-opacity vjs-opacity">
              <select>
                <option value="">---</option>
                <option value="1">Opaque</option>
                <option value="0.5">Semi-Opaque</option>
              </select>
            </span>
        </div> <!-- vjs-fg-color -->
        <div class="vjs-bg-color vjs-tracksetting">
            <label class="vjs-label">Background</label>
            <select>
              <option value="">---</option>
              <option value="#FFF">White</option>
              <option value="#000">Black</option>
              <option value="#F00">Red</option>
              <option value="#0F0">Green</option>
              <option value="#00F">Blue</option>
              <option value="#FF0">Yellow</option>
              <option value="#F0F">Magenta</option>
              <option value="#0FF">Cyan</option>
            </select>
            <span class="vjs-bg-opacity vjs-opacity">
                <select>
                  <option value="">---</option>
                  <option value="1">Opaque</option>
                  <option value="0.5">Semi-Transparent</option>
                  <option value="0">Transparent</option>
                </select>
            </span>
        </div> <!-- vjs-bg-color -->
        <div class="window-color vjs-tracksetting">
            <label class="vjs-label">Window</label>
            <select>
              <option value="">---</option>
              <option value="#FFF">White</option>
              <option value="#000">Black</option>
              <option value="#F00">Red</option>
              <option value="#0F0">Green</option>
              <option value="#00F">Blue</option>
              <option value="#FF0">Yellow</option>
              <option value="#F0F">Magenta</option>
              <option value="#0FF">Cyan</option>
            </select>
            <span class="vjs-window-opacity vjs-opacity">
                <select>
                  <option value="">---</option>
                  <option value="1">Opaque</option>
                  <option value="0.5">Semi-Transparent</option>
                  <option value="0">Transparent</option>
                </select>
            </span>
        </div> <!-- vjs-window-color -->
      </div> <!-- vjs-tracksettings -->
      <div class="vjs-tracksettings-font">
        <div class="vjs-font-percent vjs-tracksetting">
          <label class="vjs-label">Font Size</label>
          <select>
            <option value="0.50">50%</option>
            <option value="0.75">75%</option>
            <option value="1.00" selected="">100%</option>
            <option value="1.25">125%</option>
            <option value="1.50">150%</option>
            <option value="1.75">175%</option>
            <option value="2.00">200%</option>
            <option value="3.00">300%</option>
            <option value="4.00">400%</option>
          </select>
        </div> <!-- vjs-font-percent -->
        <div class="vjs-edge-style vjs-tracksetting">
          <label class="vjs-label">Text Edge Style</label>
          <select>
            <option value="none">None</option>
            <option value="raised">Raised</option>
            <option value="depressed">Depressed</option>
            <option value="uniform">Uniform</option>
            <option value="dropshadow">Dropshadow</option>
          </select>
        </div> <!-- vjs-edge-style -->
        <div class="vjs-font-family vjs-tracksetting">
          <label class="vjs-label">Font Family</label>
          <select>
            <option value="">Default</option>
            <option value="monospaceSerif">Monospace Serif</option>
            <option value="proportionalSerif">Proportional Serif</option>
            <option value="monospaceSansSerif">Monospace Sans-Serif</option>
            <option value="proportionalSansSerif">Proportional Sans-Serif</option>
            <option value="casual">Casual</option>
            <option value="script">Script</option>
            <option value="small-caps">Small Caps</option>
          </select>
        </div> <!-- vjs-font-family -->
      </div>
    </div>
    <div class="vjs-tracksettings-controls">
      <button class="vjs-default-button">Defaults</button>
      <button class="vjs-done-button">Done</button>
    </div></div></div>
        </section>
      </div>
      
    </div>
	  <div class="cta tlp">
	    <h2>TLP Guide</h2>
	      <ul class="tlps">
	        <li><span class="ihm-pill ihm-browse-4 tlp level-4">White</span>Available to anyone</li>
	        <li><span class="ihm-pill ihm-browse-3 tlp level-3">Green</span>Available to CiSP community</li>
			<li><span class="ihm-pill ihm-browse-2 tlp level-2">Amber</span>Available to specific groups of CiSP members and on a need to know basis within those members’ organisations</li>
			<li><span class="ihm-pill ihm-browse-1 tlp level-1">Red</span>Available to named individuals only</li>
		  </ul>
	  </div>
	  <div class="cta">
	    <h2>Get Involved</h2>
	    <ul>
	      <li>
	        <a class="ctaHeaderLink" href="/somethings-happening-right-now">Share an incident on CiSP</a>
	        Share the details of any suspicious activity with the community.
	      </li>
        <li>
          <a class="ctaHeaderLink" href="/ncsc-publications">NCSC publications</a>
          Get the latest publications from HMG, our analysis of the current trends, threats,
          and news for CiSP members.
        </li>
	      <li>
	        <a class="ctaHeaderLink" href="/places">Explore places and start discussions</a>
	        Join and follow topics of interest, ask questions and get answers.
	      </li>
		      <li>
		        <a class="ctaHeaderLink" href="/getting-started">Getting the most out of CiSP</a>
		        Understand how to get the most out of the platform and the community.
		      </li>
	    </ul>
	  </div>
	</div>
</div>
